src/components/inputs/SuggestSelect.jsx

```diff
@@ -387,6 +387,7 @@ const SuggestSelect = React.createClass({
 
   onOptionSelect(option, e) {
     this.selectOption(option.id);
+    if (!this.isMounted()) e.stopPropagation();
```

Comment:
Yep I did not know it was on its way to deprecation. 

I'll change this PR to use a `mounted` attribute on the class, it'll be cleaner that way.